class.js

```diff
@@ -1,20 +1,17 @@
-class Foo
-{
-color='negro';peso   =    34;
-  ver()
-  {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
+class Foo {
+  color = 'negro';
+  peso = 34;
+ver() {
+  console.info (`Color: ${this.color}. Peso: ${this.peso}`);
+  }
 }
-*/
-}
-   }    class Bar   {
-  miMetodo(){if(this.imprimir){console.info('Imprimir!');}}
+    
+class Bar {
+  miMetodo()  {
```

Comment:
Usar un espacio entre `)` y `{`:
```suggestion
  miMetodo() {
```